src/empty.js

```diff
@@ -3,7 +3,7 @@ var _hasMethod = require('./internal/_hasMethod');
 
 
 /**
- * `empty` wraps any object in an array. This implementation is compatible with the
+ * `empty` returns an empty list for any argument. This implementation is compatible with the
```

Comment:
that's definitely better, but not quite right I think. If you call `R.empty("abc")`you get back `[ ]`